src/main/java/com/cinema/Main.java

```diff
@@ -118,7 +118,7 @@ public static void main(String[] args) throws AuthenticationException {
         AuthenticationService authenticationService = (AuthenticationService) injector
                 .getInstance(AuthenticationService.class);
         authenticationService.register("roman@mail.ru", "roman");
-        log.info("Login user :" + authenticationService.login("roman@mail.ru", "roman"));
+        logger.info("Login user :" + authenticationService.login("roman@mail.ru", "roman"));
```

Comment:
let's wrap this in try-catch and add log.warn with the informative message in `catch` block